app/instance/bebbo/assets/locale/en.ts

```diff
@@ -0,0 +1,531 @@
+    homeScreenexpButton: 'Start reading',
+    homeScreentoolsHeader: 'Tools',
+    homeScreenvc: 'Vaccination',
+    homeScreenhc: 'Health check-up',
+    homeScreencg: "Child's growth",
+    actScreenheaderTitle: 'Games ',
+    actScreensugacttxt: 'Suggested games ',
+    actScreenotheracttxt: 'Other games ',
+    actScreenpendingMilestone: 'Pending',
+    actScreencompleteMilestone: 'Completed',
+    actScreentrack: 'Track',
+    actScreenmilestones: 'Milestones',
+    actScreenprematureText: 'Premature',
+    actScreenshareText: 'Share',
+    actScreenaddtoFavText: 'Save',
+    developScreenheaderTitle: 'Child development',
+    developScreenprematureText: 'Premature',
+    developScreenchartLabel: 'Pending',
+    developScreenchartText: 'Reached milestones during the current period',
+    developScreenmileStoneQ: 'Which of the milestones has your child reached?',
+    developScreentipsText: 'Tips from Doctors',
+    developScreenmileStone: 'Milestone',
+    developScreenrelatedArticleText: 'View related advice',
+    developScreenrelatedAct: 'Related games ',
+    developScreenviewDetails: 'View details',
+    articleScreenheaderTitle: 'Advice ',
+    articleScreensearchPlaceHolder: 'Search',
+    articleScreennoSearchResult: 'No advice found',
+    articleScreenshareText: 'Share',
+    articleScreenaddtoFavText: 'Add to favourites',
+    growthScreenheaderTitle: "Child's growth",
+    growthScreennoGrowthData: 'No growth measurements recorded',
+    growthScreennoGrowthDataHelpText:
+      "This will help you track your child's growth.",
+    growthScreenaddNewBtntxt: 'Add new measurement',
+    growthScreeneditNewBtntxt: 'Edit growth measurement',
+    growthScreenweightForHeight: 'Weight for height',
+    growthScreenheightForAge: 'Height for age',
+    growthScreensubHeading: 'Growth measurement',
+    growthScreenprematureText: 'Premature',
+    growthScreenlastMeasureText: 'Last measured on {{measureDate}}',
+    growthScreenallMeasurements: 'All measurements',
+    growthScreenhText: 'Height',
+    growthScreenwText: 'Weight',
+    growthScreeneditText: 'Edit',
+    growthScreendelText: 'Delete',
+    growthScreenkgText: 'kg',
+    growthScreencmText: 'cm',
+    growthScreensumHeading: 'Summary',
+    growthScreenallMeasureHeader: 'All measurements',
+    growthScreendeletebtnText: 'Delete',
+    growthScreendateMeasurementText: 'Date of measurement',
+    growthScreenenterDateMeasurementText: 'Enter date of measurement',
+    growthScreenwhereMeasured: 'Where was the child measured?',
+    growthScreendoctorMeasurePlace: "At the doctor's",
+    growthScreenhomeMeasurePlace: 'At home',
+    growthScreenenterMeasuresText: 'Enter weight and height',
+    growthScreenenterDoctorRemarkText: "Doctor's remark or comment",
+    growthScreenenterDoctorRemarkTextPlaceHolder:
+      "Enter your doctor's remark or comment",
+    growthScreensaveMeasures: 'Save',
+    growthScreensaveMeasuresDetails: 'Save',
```

Comment:
Maybe we can replcae growthScreensaveMeasuresDetails with growthScreensaveMeasures in the code.